Modules/posixmodule.c

```diff
@@ -2787,8 +2790,16 @@ os_chmod_impl(PyObject *module, path_t *path, int mode, int dir_fd,
     else
 #endif
 #ifdef HAVE_LCHMOD
-    if ((!follow_symlinks) && (dir_fd == DEFAULT_DIR_FD))
+    if ((!follow_symlinks) && (dir_fd == DEFAULT_DIR_FD)) {
         result = lchmod(path->narrow, mode);
+        /*
+         * similar to the comment below about fchmodat(), some platforms
+         * (for instance musl libc) do not ship a functional lchmod().
+         */
+         lchmod_unsupported =
+            result &&
+            ((errno == ENOTSUP) || (errno == EOPNOTSUPP));
```

Comment:
Oh, there are two different error codes? Which one is raised on muslc?

Reply:
muslc raises `EOPNOTSUPP` (errno 95) though on my platform they were `#define`d to the same thing.  I simply picked the same errnos as `fchmodat` catches below -- I suspect implementations use `fchmodat` to implement `lchown` (or the other way around).